projects/power_distribution/inc/output.h

```diff
@@ -105,6 +105,7 @@ typedef struct OutputBts7200Spec {
 typedef struct OutputBts7040Spec {
   Pca9539rGpioAddress enable_pin;
   uint8_t mux_selection;  // what should we select on the mux to read current from the BTS7040?
+  bool use_bts7004_scaling;
```

Comment:
In output_config.c, in the spec for `FRONT_OUTPUT_UV_VBAT` (around line 300 or so), could you set this attribute to `true`? UV VBAT is on a BTS7004, and changing its scaling was the reason we needed this ticket.